docs/DeveloperGuide.md

```diff
@@ -462,23 +563,36 @@ Use case ends.
 
 Use case ends.
 
-**Use case 8: Filter order list by order status**
+
```

Comment:
I mean to reder to line 560, since precondition already states a displayed order list, consider removing line 560 as per previous use cases.

Reply:
Although that might mess with ordering so i suggest just replace it without the UC04